CMakeLists.txt

```diff
@@ -161,3 +161,7 @@ if (TARGET visrtxCompileShaders AND VISRTX_PRECOMPILE_SHADERS)
     "execute_process(COMMAND ${CMAKE_BINARY_DIR}/visrtxCompileShaders)"
   )
 endif()
+
+project(pdb_reader)
+
+add_executable(pdb_reader main.cpp)
```

Comment:
Didn't catch this right away -- this looks like some stray lines left in there by accident?

Reply:
No idea where this is coming from... Sorry... maybe some automatic auto-completion